api/apps/api/src/modules/scenarios/scenarios.service.ts

```diff
@@ -112,6 +112,10 @@ import { InjectEntityManager } from '@nestjs/typeorm';
 import { apiConnections } from '@marxan-api/ormconfig';
 import { EntityManager } from 'typeorm';
 import { UserId } from '@marxan/domain-ids';
+import {
+  DeleteScenario as ScenarioUnusedResources,
```

Comment:
picky detail - I am unclear about this import alias... maybe you meant `DeleteScenarioUnusedResources`? but even then, I am not sure I understand the rationale (also, it's late on a Friday afternoon, so I may not understanding *anything*)

Reply:
(especially because we do have a `ScenarioUnusedResources` class, in any case) 🤔